Jenkinsfile

```diff
@@ -0,0 +1,3 @@
+DEFAULT_NODE_VERSION = '14'
+NODE_VERSIONS = ['8', '14']
```

Comment:
10? 12?

Reply:
I thought it will take longer, 10-12 should be covered with 14 :D